ruby/vampires.rb

```diff
@@ -0,0 +1,65 @@
+# Release 3
+puts "How many employees will be interviewed?"
+number_of_employees = gets.chomp.to_i
+interview = 1
+
+while interview <= number_of_employees
+
+# Release 1
+  puts "What is your name?"
+  name = gets.chomp.to_s
+
+  puts "How old are you?"
+  age = gets.chomp.to_i
+
+  puts "What year were you born?"
+  year = gets.chomp.to_i
+
+  real_age = Time.now.year - year
+
+# Added a loop to both garlic and insurance questions to ensure only two possible answers.
+  puts "Our company cafeteria serves garlic bread. Should we order some for you? Type 'y' or 'n'."
+  garlic_bread = gets.chomp.downcase
+  until garlic_bread == "y" || garlic_bread == "n"
+    puts "Our company cafeteria serves garlic bread. Should we order some for you? Type 'y' or 'n'."
+    garlic_bread = gets.chomp.downcase
+  end
+
+  puts "Would you like to enroll in the company's health insurance? Type 'y' or 'n'."
+  insurance = gets.chomp.downcase
+  until insurance == "y" || insurance == "n"
+    puts "Would you like to enroll in the company's health insurance? Type 'y' or 'n'."
+    insurance = gets.chomp.downcase
+  end
+
+# Release 2
+  if (real_age == age) && (garlic_bread == "y" || insurance == "y")
```

Comment:
Consider nesting your control flow.  In this case if the we know the employee's name is Drac of Fang then we don't need to run anymore of these conditions. 

```
if name == "Drake Cula" || name == "Tu Fang"
  results = "Definitely a vampire"
else
  # run other checks
end
```